demo/src/test/scala/dev/atedeg/ecscalademo/systems/VelocityEditingSystemTest.scala

```diff
@@ -1,34 +1,48 @@
 package dev.atedeg.ecscalademo.systems
 
+import dev.atedeg.ecscala.World
 import dev.atedeg.ecscala.dsl.ECScalaDSL
 
 import scala.language.implicitConversions
 import dev.atedeg.ecscala.util.types.given
-import dev.atedeg.ecscalademo.given
+import dev.atedeg.ecscalademo
 import dev.atedeg.ecscalademo.fixtures.VelocityFixture
-import dev.atedeg.ecscalademo.{ MouseState, PlayState, Point, Velocity }
+import dev.atedeg.ecscalademo.{ MouseState, PlayState, Point, State, Velocity }
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 
-class VelocityEditingSystemTest extends AnyWordSpec with Matchers with ECScalaDSL {
+class VelocityEditingSystemTest extends AnyWordSpec with Matchers {
 
   "A VelocityEditingSystem" should {
     "run when the simulation is not running, there is a selected ball and the game is in velocity editing mode" in new VelocityFixture {
       system.shouldRun shouldBe false
-      PlayState.playing = false
-      PlayState.velocityEditingMode = true
-      PlayState.selectedBall = Some(entity1)
+      playState.gameState = State.ChangeVelocity
+      playState.selectedBall = Some(entity1)
+      mouseState.clicked = true
       system.shouldRun shouldBe true
     }
-    "correctly update the velocity" in testNewExpectedVelocity(Point(1, 2), Velocity(2, 4))
-    "limit the maximum velocity" in testNewExpectedVelocity(Point(3000, 0), Velocity(2000, 0))
+    "correctly update the velocity" in new VelocityFixture {
+      testNewExpectedVelocity(world, system, playState, mouseState, Point(1, 2), Velocity(2, 4))
+    }
+    "limit the maximum velocity" in new VelocityFixture {
+      testNewExpectedVelocity(world, system, playState, mouseState, Point(3000, 0), Velocity(2000, 0))
+    }
   }
 
-  def testNewExpectedVelocity(mouseCoordinates: Point, expectedVelocity: Velocity): Unit = new VelocityFixture {
-    PlayState.selectedBall = Some(entity1)
-    MouseState.clicked = true
-    MouseState.coordinates = mouseCoordinates
-    world.update(10)
+  private def testNewExpectedVelocity(
+      wrld: World,
+      sys: VelocityEditingSystem,
+      pState: PlayState,
+      mState: MouseState,
```

Comment:
I really can't stand abbreviations
```suggestion
      world: World,
      system: VelocityEditingSystem,
      playState: PlayState,
      mouseState: MouseState,
```

Reply:
Those abbreviation in needed due a clash of name in the fixture. If clash happen the test fail